doc/api/dynakube-api-ref.md

```diff
@@ -40,6 +40,13 @@
 |tolerations|Set tolerations for the ActiveGate pods|-|array|
 |topologySpreadConstraints|Adds TopologySpreadConstraints for the ActiveGate pods|-|array|
 
+### .spec.metaDataEnrichment
+
+|Parameter|Description|Default value|Data type|
+|:-|:-|:-|:-|
+|enabled|Enables MetaDataEnrichment Enabled by default.|-|boolean|
```

Comment:
the default value is known:
```
|enabled|Enables MetaDataEnrichment Enabled by default.|true|boolean|
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2998/commits/377ee7de25d0ef6e3aaa1218f47834a1536c95e2